tests/Properties.hs

```diff
@@ -464,6 +464,17 @@ short_tests =
     , testProperty "pinned"                   prop_short_pinned
     ]
 
+------------------------------------------------------------------------
+-- Strictness checks.
+
+explosiveTail :: L.ByteString -> L.ByteString
+explosiveTail = (`L.append` error "Tail of this byte string is undefined!")
+
+explosiveTailD :: D.ByteString -> D.ByteString
```

Comment:
`L.ByteString` and `D.ByteString` are the same type, so you can use the same `explosiveTail` for both.

Reply:
True!